### PR TITLE
Switching between food clustering algorithms

### PR DESCRIPTION
**Pull request by ksofiyuk**
Issue #122 and discussion from PR #114 
## Description

I implement possibility to switch between two clustering approaches (chancity and ksofiyuk) by pressing key 'F'.
## Motivation and Context

In order to compare different approaches. Experiments have shown that clustering algorithm does not cause the lags (nor mine, nor chancity) in most cases. 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: bot.user.js [hunk 1]
```diff
@@ -511,18 +566,29 @@ var bot = (function() {
...
 
-                    window.sortedFood = bot.getSortedFood();
-                    window.currentFood = window.sortedFood[0];
+                    var accelerationClusterSize;
+                    if (bot.foodClusteringMode == 1) {
+                        bot.computeFoodGoal();
+                        accelerationClusterSize = 120;
+                    } else {
+                        var foods = bot.getFood();
+                        window.sortedFood = foods.map(bot.foodClusters).sort(bot.sortFoodClusters);
```
Comment (ksofiyuk) [inline bot.user.js]: > Line is too long.
> Too many errors. (58% scanned).

What does it mean?

Comment (tjorim) [inline bot.user.js]: I assume Hound means the .sort of a .map of foods, but it looks good to me though.

**Comment by ermiyaeskandary** [comment]
:shipit

**Comment by ermiyaeskandary** [comment]
Approved 

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/124/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Comment by ermiyaeskandary** [comment]
Needs to be rebased...

**Comment by ChadSki** [comment]
Approved :+1:

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/124/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Pull request merged**